index.css

```diff
@@ -607,30 +608,40 @@ label {
     line-height: 60px;
     letter-spacing: 0.37px;
     font-weight: 700;
-    width: 420px;
+    width: 90%;
     padding-bottom: 10px;
+    padding-left: 260px;
+    padding-right: 260px;
   }
 
   .section p {
     color: #4d5c76;
-    font-size: 16px;
+    font-size: 20px;
     font-size: 1.75vw;
-    line-height: 24px;
+    line-height: 28px;
     letter-spacing: 0;
-    width: 620px;
-    font-weight: 500;
-    align-items: left;
+    width: 90%;
+    font-weight: 400;
+    padding-left: 260px;
+    padding-right: 260px;
   }
 
   .box1 {
     background-image: url("images/Header bg.png");
-    background-size: contain;
     background-repeat: no-repeat;
     width: 100%;
     margin: auto;
     height: 440px;
     display: block;
+  }
+
+  .social-media {
+    display: block;
     justify-content: center;
+    width: 90%;
+    margin: auto;
+    padding-left: 120px;
+    padding-right: 120px;
   }
 
```

Comment:
:red_circle: [OPTIONAL] Please make sure your desktop version doesn't affect the mobile version of your portfolio. As for now, the mobile version has been affected immensely. Kindly fix this to have a professional portfolio. I think the big issue is with your about section The language, and skills framework section should move beneath the about block on the mobile version.

![Peek 2023-01-21 23-17](https://user-images.githubusercontent.com/26736582/213887487-26d72dce-972e-4a54-ac11-6bfd17a0dbc5.gif)